app/models/concerns/awardee.rb

```diff
@@ -0,0 +1,31 @@
+module Awardee
+  def acquired_medals(organization=Organization.current)
+    @acquired_medals ||= medals_for (completed_content_in organization)
+  end
+
+  def unacquired_medals(organization=Organization.current)
+    @unacquired_medals ||= medals_for (uncompleted_content_in organization)
+  end
+
+  private
+
+  def medals_for(content)
+    content.map(&:medal)
+  end
+
+  def awardable_contents_in(organization)
+    @awardable_contents_in ||= organization.gamification_enabled? ? organization.all_contents.select(&:medal_id) : []
```

Comment:
We could move this to organization and make it just organization.awardable_contents_for(self). What do you think?